File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int p = 0;
        int y = 0;
        String str = s.toLowerCase();
        for(char x:str.toCharArray()){
            if (x=='p') p++;
            else if (x=='y') y++;
        }

        return (p == y ? true : false);
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.swift
import Foundation

func solution(_ s:String) -> Bool
{
    var lowercasedString = s.lowercased()

    var p_count = lowercasedString.filter{ $0 == "p"}.count
    var y_count = lowercasedString.filter{ $0 == "y"}.count   

    
    return p_count == 0 && y_count == 0 ? true : p_count == y_count ? true : false
}